### PR TITLE
Use pkg/errors for custom errors and wrapping message

### PR DESCRIPTION
**Pull request by sdurrheimer**
+ display stackstrace with the `verbose` flag.

### CODE REVIEW

File: cmd/promu.go [hunk 1]
```diff
@@ -54,8 +55,7 @@ var Promu = &cobra.Command{
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
 	if err := Promu.Execute(); err != nil {
-		fmt.Println(err)
```
Comment (grobie) [inline cmd/promu.go]: Who prints the error message now??

Comment (sdurrheimer) [inline cmd/promu.go]: Strangely cobra was doing it by itself. Removing this line did fixed the double error display.

**Comment by grobie** [comment]
:/ not a fan of implicit "magic" code. The beauty of go is that code is
usually easily understandable even if you're not familiar with a library.

:+1: then

**Comment by sdurrheimer** [comment]
I agree, I just didn't found how to prevent cobra from doing it, in that way I could display it myself. I will double check shortly.

**Comment by sdurrheimer** [comment]
@grobie Looks like there is a way to silence the [cobra Execute() error printing](https://sourcegraph.com/github.com/spf13/cobra@8d4ce3549a0bf0e3569df3aae7423b7743cd05a9/-/blob/command.go?utm_source=chrome-extension#L720-721).

**Comment by sdurrheimer** [comment]
@grobie Updated

**Comment by sdurrheimer** [comment]
Renamed `er` as `printErr`

**Pull request merged**